app/models/support_interface/providers_filter.rb

```diff
@@ -105,6 +112,17 @@ def filter_records(providers)
         providers = providers.left_outer_joins(:provider_permissions).where(provider_users_providers: { id: nil })
       end
 
+      if applied_filters[:accredited_provider].present?
+        accrediting_provider_ids = Provider
+                                    .where("CONCAT(providers.name, ' ', providers.code) ILIKE ?", "%#{applied_filters[:accredited_provider]}%")
+                                    .map(&:id)
+
+        providers = providers
+                    .distinct
+                    .joins(courses: :accredited_provider)
```

Comment:
Does this need to join to `accredited_provider` as the condition below is on `courses.accredited_provider_id`?

Reply:
yea good point! Will sort now